devlib/native/win_native.cpp

```diff
@@ -212,7 +212,8 @@ namespace winutil {
                 continue;
             }
 
-            deviceHandler(detailData);
+            if (deviceHandler(detailData))
```

Comment:
You have two different aliases: `DeviceHandler` and `DeviceInterfaceHandler`
https://github.com/emlid/devlib/blob/e34e5ec12b6773e61b91a96175b7b76d62c93c28/devlib/native/win_native.cpp#L100-L101
And then you (or maybe not you, I haven't seen `git blame` though) name a variable `deviceHandler` of type `DeviceInterfaceHandler`, which is a bit confusing IMO, don't you think?

Reply:
Changed